spot-contracts/contracts/FeePolicy.sol

```diff
@@ -96,21 +96,21 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
 
     /// @dev NOTE: We updated the type of the parameters from int256 to uint256, which is an upgrade safe operation.
     struct RolloverFeeParams {
-        /// @notice The maximum debasement rate for perp,
-        ///         i.e) the maximum rate perp pays the vault for rollovers.
+        /// @notice The minimum rollover fee percentage enforced by the contract.
+        /// @dev This is represented as signed fixed point number with {DECIMALS} places.
+        ///      The rollover fee percentage returned by the fee policy will be no lower than
+        ///      this specified value.
```

Comment:
Maybe add a comment that this effectively acts as a maximum debasement amount (when negative).